src/main/java/org/mule/extension/sftp/api/AbstractFileAttributes.java

```diff
@@ -84,6 +84,14 @@ public void setName(String fileName) {
     this.fileName = fileName;
   }
 
+  public String getFileName() {
```

Comment:
Are these methods are covered in munit? Also can you add Junit test for this as these are public methods.

Reply:
munits directly use the attribute name itself which is already filename. Added junit to cover these public methods.